Source/WebCore/editing/VisibleSelection.cpp

```diff
@@ -294,7 +294,7 @@ void VisibleSelection::adjustSelectionRespectingGranularity(TextGranularity gran
             VisiblePosition wordEnd(endOfWord(originalEnd, side));
             VisiblePosition end(wordEnd);
             
-            if (isEndOfParagraph(originalEnd) && !isEmptyTableCell(m_start.deprecatedNode())) {
+            if (isEndOfParagraph(originalEnd) && !isEmptyTableCell(RefPtr { m_start.deprecatedNode() }.get())) {
```

Comment:
Since a `Position` holds a `RefPtr` to a node we should have a function that can take advantage of that instead of churning just to read a member in a case like this?